README.md

```diff
@@ -97,4 +99,4 @@ EXAMPLES:
 
 
 ## Contributing
-If anything feels off, or if you feel that some functionality is missing, please check out the [contributing page](CONTRIBUTING.md). There you will find instructions for sharing your feedback, building the tool locally, and submitting pull requests to the project.
\ No newline at end of file
+If anything feels off, or if you feel that some functionality is missing, please check out the [contributing page](CONTRIBUTING.md). There you will find instructions for sharing your feedback, building the tool locally, and submitting pull requests to the project.
```

Comment:
There is no change here right?

Reply:
No change. Could be a minor formatting by the editor.